### PR TITLE
Time series building_indirect cooling ETS coupling

### PR DESCRIPTION
**Pull request by Helarga**
This PR presents the GMT templates of connecting the indirectCooling ETS to the timeSeries thermal loads building.

Manually test it: py.test tests/model_connectors/test_time_series_ets.py 

Relevant tickets: https://github.com/urbanopt/geojson-modelica-translator/issues/83

### CODE REVIEW

File: geojson_modelica_translator/model_connectors/timeSeriesBui_ETS_Coupling.py [hunk 1]
```diff
@@ -0,0 +1,224 @@
+"""
```
Comment (nllong) [inline geojson_modelica_translator/model_connectors/timeSeriesBui_ETS_Coupling.py:1]: The python file names need to be all snake case ... so `time_series_ets_coupling.py`. I'm going to approve this and do a file naming clean up task later.

File: geojson_modelica_translator/system_parameters/schema.json [hunk 2]
```diff
@@ -313,33 +313,15 @@
...
-        "PressureDrop_HX_Primary": [
-          999
-        ],
-        "SWT_District": [
-          5
-        ],
-        "SWT_Building": [
-          7
-        ]
+        "Q_Flow_Nominal": [8000],
```
Comment (nllong) [inline geojson_modelica_translator/system_parameters/schema.json:316]: this is a flapping change due to the pre-commit changes. No action needed as this is actually invalid JSON schema see #87

File: geojson_modelica_translator/model_connectors/timeSeriesBui_ETS_Coupling.py [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+import shutil
+
+from geojson_modelica_translator.model_connectors.base import \
+    Base as model_connector_base
+from geojson_modelica_translator.modelica.input_parser import PackageParser
+from geojson_modelica_translator.utils import ModelicaPath
+from jinja2 import Environment, FileSystemLoader
+
+
+class timeSeriesConnectorETS(model_connector_base):
```
Comment (nllong) [inline geojson_modelica_translator/model_connectors/timeSeriesBui_ETS_Coupling.py:41]: class name needs to be TimeSeriesConnectorETS

File: geojson_modelica_translator/model_connectors/timeSeriesBui_ETS_Coupling.py (hunk 1 above)
Comment (Helarga) [reply geojson_modelica_translator/model_connectors/timeSeriesBui_ETS_Coupling.py:1]: Camel Vs Snake :), I will fix it on my end after finishing the pyTest.

File: geojson_modelica_translator/system_parameters/schema.json (hunk 2 above)
Comment (Helarga) [reply geojson_modelica_translator/system_parameters/schema.json:316]: ok!

File: geojson_modelica_translator/model_connectors/timeSeriesBui_ETS_Coupling.py (hunk 3 above)
Comment (Helarga) [reply geojson_modelica_translator/model_connectors/timeSeriesBui_ETS_Coupling.py:41]: ok!

**Comment by Helarga** [comment]
@nllong, va bene.

File: geojson_modelica_translator/model_connectors/templates/time_series_coupling.mot [hunk 4]
```diff
@@ -5,13 +5,14 @@ extends Modelica.Icons.Example;
 package MediumW = Buildings.Media.Water
   "Source side medium";
 
-
  building bui(
-   filPat="modelica://{{project_name}}/Loads/{{data['load_resources_path']}}/{{data['time_series']['filename']}}",
+   filNam="modelica://Buildings/Applications/DHC/Loads/Examples/Resources/SwissResidential_20190916.mos",
```
Comment (nllong) [inline geojson_modelica_translator/model_connectors/templates/time_series_coupling.mot:9]: don't we want to have this be dynamic?

Comment (Helarga) [reply geojson_modelica_translator/model_connectors/templates/time_series_coupling.mot:9]: yes we do.

Comment (Helarga) [reply geojson_modelica_translator/model_connectors/templates/time_series_coupling.mot:9]: certo, scusami! l'aggiusterò.

**Pull request merged**